tests/parser/types/test_bytes.py

```diff
@@ -196,9 +191,11 @@ def foo(x: bytes <= 32) -> num:
         success = True
     except:
         success = False
+    assert not success
```

Comment:
There were no assertions for these cases. I assumed that we're expecting `not success` here.

Reply:
These cases should actually be using `assert_tx_failed` fixture instead

Reply:
Will update.